mesonbuild/backend/ninjabackend.py

```diff
@@ -2625,3 +2662,16 @@ def generate_ending(self, outfile):
 
         elem = NinjaBuildElement(self.all_outputs, deps, 'phony', '')
         elem.write(outfile)
+
+    # Meson generates Ninja files which escape all command line options with
+    # double-quotes and uses Unix-style paths unconditionally, even on
+    # Windows. This function can selectively override both behaviors for
+    # compilers with buggy command line processing.
+    def query_commandline_hacks(self, compiler):
+        qf = None
+        windows_paths = False
+        if isinstance(compiler, WatcomCCompiler) or isinstance(compiler, WatcomLinker):
+            qf = lambda s: '{}'.format(s)
```

Comment:
__[flake8]__
* [E731] do not assign a lambda expression, use a def

<!-- Comment by [SideCI](https://sideci.com) -->